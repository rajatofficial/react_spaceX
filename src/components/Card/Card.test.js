import React from 'react';
import { shallow } from 'enzyme';
import SatelliteCard from './satelliteCard';

describe('Satellite Card Component', () => {
  it('Satellite Card should render correctly in "debug" mode', () => {
    const component = shallow(<SatelliteCard debug />);
  
    expect(component).toMatchSnapshot();
  });
});