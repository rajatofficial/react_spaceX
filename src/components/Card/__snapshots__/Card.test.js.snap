// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Satellite Card Component Satellite Card should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
